.github/issue_template.md

```diff
@@ -0,0 +1,58 @@
+---
+name: Issue Report
+about: Create a report to report a bug, ask a question, or provide feedback
+title: "[BUG/QUESTION/SUGGESTION] Describe the issue in a concise title"
+labels: ''
+assignees: ''
+
+---
+
+# **Please select the type of issue you are creating:**
+
+- [ ] Bug Report
+- [ ] Question
+- [ ] Suggestion / Enhancement Request
```

Comment:
We can actually add separate templates for each of these instead, see https://github.com/python/cpython/tree/7316dfb0ebc46aedf484c1f15f03a0a309d12a42/.github/ISSUE_TEMPLATE for example.

Reply:
@zware that's awesome I'll definitely break those up!

Reply:
Done!